frontend/src/components/common/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import styles from './styles.module.scss';
+
+type Props = {
+  id: string;
+  icon: string;
```

Comment:
```suggestion
  icon: IconName;
```

Reply:
Because of the impossibility to use img as enum, I made it a constraint and left string in type
src attribute needs a string